android/library/src/main/java/com/liferay/mobile/screens/auth/login/LoginScreenlet.java

```diff
@@ -207,7 +210,10 @@ protected View createScreenletView(Context context, AttributeSet attributes) {
 
 	@Override
 	protected BaseLoginInteractor createInteractor(String actionName) {
-		if (BASIC_AUTH.equals(actionName)) {
+		if (authenticationType.equals(AuthenticationType.COOKIE)) {
+			return new LoginCookieInteractor();
+		}
+		else if (BASIC_AUTH.equals(actionName)) {
```

Comment:
@victorg1991 the action name should be always the same, and we'll rely on "authenticationType" to decide the interactor to be created.
Check iOS as well